packetfence.rsyslog

```diff
@@ -0,0 +1,103 @@
+# RADIUS
+if $syslogtag contains "auth" and $syslogfacility-text == "local1" then -/usr/local/pf/logs/radius.log
```

Comment:
AHHHH, I think I get it about what I said above

But, is there a way to accomplish the same thing without doing the local changing dance ?